vhdbuilder/packer/backfill-cleanup.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -x
+
+EXPIRATION_IN_HOURS=168
+# convert to seconds so we can compare it against the "tags.now" property in the resource group metadata
+(( expirationInSecs = ${EXPIRATION_IN_HOURS} * 60 * 60 ))
+# deadline = the "date +%s" representation of the oldest age we're willing to keep
+(( deadline=$(date +%s)-${expirationInSecs%.*} ))
+
+if [[ "${MODE}" != "windowsVhdMode" ]]; then
+  exit 0
+fi
+
+# we will clean old sig image-definitions/versions/galleries, RGs only if 
+# we have overriden the pipeline variable SKIP_IMAGE_DELETION_BACKFILL to False
```

Comment:
not necessary, but I personally think this entire flow would be a lot more readable if we renamed SKIP_IMAGE_DELETION_BACKFILL to something like PERFORM_RESOURCE_DELETION_BACKFILL and swap around the boolean values - generally better to not have negatives/negations in the names of booleans